extensions/azure-app-configuration/runtime/src/main/java/io/quarkiverse/azure/app/configuration/AzureAppConfigurationConfigSourceFactory.java

```diff
@@ -38,6 +41,7 @@ private Map<String, String> getAzureAppConfiguration(final AzureAppConfiguration
         VertxAsyncHttpClientBuilder httpClientBuilder = new VertxAsyncHttpClientBuilder().vertx(vertx);
 
         ConfigurationClientBuilder clientBuilder = new ConfigurationClientBuilder()
+                .clientOptions(new ClientOptions().setApplicationId(AZURE_QUARKUS_APP_CONFIGURATION))
```

Comment:
[AzureSpringIdentifier](https://github.com/Azure/azure-sdk-for-java/blob/2c68397adca8db969f35d5787d5c906213d164bc/sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/util/AzureSpringIdentifier.java#L52C70-L52C94) has version tag, do we need version here?

```
    public static final String AZURE_SPRING_KEY_VAULT_CERTIFICATES = "az-sp-kv-ct/" + VERSION;
```

Reply:
I would like to see a comment near this constant declaration that shows a reader where to find the definitive registry of such identifiers. Similar to how we have defined a list of `ms.custom` tags in our docs. Surely this list must exist in some central location, right?